ticdc/ticdc-changefeed-config.md

```diff
@@ -169,6 +169,8 @@ enable-partition-separator = false
 # null = '\N'
 # Whether to include commit-ts in CSV rows. The default value is false.
 # include-commit-ts = false
+# The encoding method of binary data, which can be 'base64' or 'hex'. The default value is 'base64'.
+# binary-encoding-method = 'base64'
```

Comment:
```suggestion
# binary-encoding-method = 'base64' (new in v6.5.4)
```

Reply:
sync changes in https://github.com/pingcap/docs-cn/pull/14707/commits/d78063bfb7219e5053d7cb6977d91f8bd95906ad